### ISSUE TITLE
CMake warning FPHSA name mismatch [8450]

### ISSUE THREAD

**Issue by dirk-thomas** [eProsima/Fast-RTPS#1216]
Building (the 2.0.x branch) with CMake 3.17+ results in a CMake warning about a name mismatch in the TinyXML2 module:

```
CMake Warning (dev) at /usr/local/Cellar/cmake/3.17.2/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:272 (message):
  The package name passed to `find_package_handle_standard_args` (tinyxml2)
  does not match the name of the calling package (TinyXML2).  This can lead
  to problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/modules/FindTinyXML2.cmake:45 (find_package_handle_standard_args)
  cmake/common/eprosima_libraries.cmake:96 (find_package)
  CMakeLists.txt:221 (eprosima_find_thirdparty)
```

**Comment by MiguelCompany** [comment]
@dirk-thomas This should have been fixed by #1217

**Comment by dirk-thomas** [comment]
Thanks.
